test/python/test_status_api.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
+import unittest
+
+import openapi_client
+
+from . import api_config
+
+AUTHENTICATOR_FIELDS = [
+    "enabled",
+    "installed",
+    "configured",
+]
+
+class TestAuthenticatorsApi(api_config.ConfiguredTest):
+    """AuthenticatorsApi unit test stubs"""
+    def setUp(self):
+        self.api = openapi_client.api.status_api.StatusApi(self.client)
+
+    def test_authenticators(self):
+        """Test case for authenticators
+
+        """
+        authenticators = self.api.authenticators()
+
+        self.assertIsInstance(authenticators, openapi_client.models.authenticators.Authenticators)
+
+        for i in AUTHENTICATOR_FIELDS:
```

Comment:
Let's add a descriptive comment making it clear what we're testing here
We'll also need a test case for the `UnauthorizedError`

Reply:
Turns out you actually dont need authorization for that endpoint. I removed 401 from the spec